clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -56,12 +56,33 @@ trait IoUtil {
     gs.exists(path) == 0
   }
 
+  def getMd5HashOfGoogleObject(path: String): String = {
```

Comment:
Why isn't this in a `GsUtil` companion object, e.g. `GsUtil.md5Hash()`?

Reply:
Defining it in the `GsUtil` class makes it simpler to mock (which the backfill unit tests do).